doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -71,13 +71,23 @@ Some of these can be configured using the Kconfig options listed below:
 * :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` enables an optional rotating device identifier feature that provides an additional unique identifier for each device.
   This identifier is similar to the serial number, but it additionally changes at predefined times to protect against long-term tracking of the device.
 
-Amazon FFS support
-==================
+Amazon Frustration-Free Setup support
+=====================================
 
-Matter in the |NCS| supports `Amazon Frustration-Free Setup (FFS)`_ that allows Matter devices to be automatically commissioned to the Matter network using the Matter-enabled Amazon Echo device.
+Frustration-Free Setup (FFS) is Amazon's proprietary technology that uses the user's Amazon account and Alexa services to register an IoT device and provision it to a mesh network.
+It does not replace the default commissioning process, but uses the cloud-based Amazon Device Setup Service to act as a middleman between the IoT devices that want to join a network and their provisioners.
```

Comment:
What is a provisioner here - a person or a commissioner device? I'm not sure if naming it a middleman is correct. It reminds me something that stands between commissioner and commissionee to somehow direct the information to them. In case of Amazon commissioner and commissionee acts in the same way, as for other Matter ecosystems. The difference is a way that commissioner gets information necessary for commissioning. Traditionally the user scans some QR code or enters pin code. For Amazon commissioner just asks cloud for data.